README.md

```diff
@@ -37,6 +37,8 @@ Deploy Claudie Kubernetes [manifests](https://github.com/Berops/claudie/tree/mas
 kustomize build | kubectl apply -f -
```

Comment:
I think this would be nice too:

Replacing
```
Deploy Claudie Kubernetes [manifests](https://github.com/Berops/claudie/tree/master/manifests/claudie) into a Kubernetes cluster.
```
by
```
Deploy Claudie Kubernetes [manifests/claudie](https://github.com/Berops/claudie/tree/master/manifests/claudie) into a Kubernetes cluster.
```